src/main/java/testproject/TestProject.java

```diff
@@ -1,11 +1,19 @@
 package testproject;
 
+import java.math.BigDecimal;
+
 public class TestProject {
 	
 	public static void main(String[] args) {
 		String alternativeName = "Max";
+		try {
+			BigDecimal youShouldNotDoThis = new BigDecimal(1.5d);
+		} catch (Throwable throwable) {
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png 'Severity: BLOCKER') Catch Exception instead of Throwable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.rz.is/coding_rules#rule_key=squid%3AS1181)